crates/e2e/src/setup/colocation.rs

```diff
@@ -122,6 +123,7 @@ endpoint = "{endpoint}"
 relative-slippage = "0.1"
 account = "{account}"
 merge-solutions = {merge_solutions}
+quote-using-limit-orders = {quote_using_limit_orders}
```

Comment:
Since I am trying to use [the same solution](https://github.com/cowprotocol/services/blob/2ace2eb512241c80d4115e539b98921756479507/crates/e2e/tests/e2e/cow_amm.rs#L903) for both `/quote` and `/solve`, this config is required to satisfy [this](https://github.com/cowprotocol/services/blob/a17e4196293a7814d3d9bee90dec31ad9860bd14/crates/driver/src/domain/competition/solution/trade.rs#L163-L168) validation.